ms2/src/org/labkey/ms2/protein/tools/GoLoader.java

```diff
@@ -90,7 +102,7 @@ public static WebPartView getCurrentStatus(String message)
 
     public static GoLoader getFtpLoader()

```

Comment:
Not a big deal, but change to `getHttpLoader()`?

Reply:
Changed.